todocore/src/main/resources/application.yml

```diff
@@ -7,7 +7,7 @@ spring:
   datasource:
     driverClassName: org.postgresql.Driver
     username: postgres
-    password: postgres
+    password: 123
```

Comment:
you shouldn't update it I suppose, it's your local variables. The rule here is that you shouldn't update anything if it's not necessary.